### PR TITLE
trying to fix travis/docker

### PR DESCRIPTION
**Pull request by NunoDasNeves**
hnng

### CODE REVIEW

File: .coveralls.yml [hunk 1]
```diff
@@ -1 +1 @@
-repo_token: D402uNrUOqV4daKETO1WGsjoD7DrJyhaV
+repo_token: $COVERALLS_TOKEN
```
Comment (DarkPurple141) [inline .coveralls.yml]: This works? This is a yml tho?

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1 @@
+dummy
```
Comment (DarkPurple141) [inline public/index.html]: why do u need this again?

File: .coveralls.yml (hunk 1 above)
Comment (NunoDasNeves) [reply .coveralls.yml]: it does not. trying to fix rn

File: public/index.html (hunk 2 above)
Comment (NunoDasNeves) [reply public/index.html]: there's an api test that serves index.html

Comment (NunoDasNeves) [reply public/index.html]: added a spicy comment in there to explain

**Pull request merged**